android/src/main/res/values/strings.xml

```diff
@@ -14,4 +14,5 @@
     <string name="submit_button">Submit</string>
     <string name="nps_not_at_all_likely">Not at all Likely</string>
     <string name="nps_extremely_likely">Extremely Likely</string>
+    <string name="thanks_message">Thanks for taking the survey</string>
```

Comment:
`.` :trollface: 

![image](https://user-images.githubusercontent.com/6950766/208823750-ea02aea7-8642-4de1-a274-df051fe22549.png)

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/124/commits/3b7b880311e92f9501eb7218bc517f9bcd6f6112